rust/Cargo.toml

```diff
@@ -26,15 +26,19 @@ rand = "0.7"
 rand_chacha = "0.2.1"
 rayon = "1.2.1"
 anyhow = "1.0.23"
-bellperson = { version = "0.8.0", features = ["gpu"] }
+bellperson = { version = "0.9.0", features = ["gpu"] }
 serde_json = "1.0.46"
 
 [dependencies.filecoin-proofs-api]
-package = "filecoin-proofs-api"
-version = "4.0.0"
+git = "https://github.com/filecoin-project/rust-filecoin-proofs-api"
+branch = "master"
+
+#[dependencies.filecoin-proofs-api]
```

Comment:
Question: What was the rationale for pulling this stuff in from Git instead of crates.io?

Reply:
I just wanted to point master to `-api` master for now.  That will go away when we have a new point release.

Reply:
Back story -- right this moment, -api has some perf changes in master that points to proofs master, and we want some testing done that needs ffi to be buildable, so this is a way to go about it.